.github/workflows/make-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: make build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  web-e2e:
+    name: Frontend e2e test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: 14.x
+
+      - name: Setup golang environment
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: '1.13'
+
+      - name: make build
+        run: make build
```

Comment:
How to check if build successfully?

Reply:
IMO. Can we put this command at the front e2e test?
Start the manger-api by the binary instead of `go run`

Reply:
If an error occurs, it will fail

Reply:
In fact, I did think about it, but cypress is disabled in make build

> IMO. Can we put this command at the front e2e test?
> Start the manger-api by the binary instead of `go run`